src/test/fixtures/config/configuration-file.tsx

```diff
@@ -0,0 +1,81 @@
+import { join } from "path";
+import {
+  didCredential,
+  localCredential,
+  rinkebyCredential,
+  ropstenAWSCredential,
+  ropstenCredential,
+} from "./credential";
+import { documentFormsV2 } from "./forms/v2";
+import { documentFormsV3 } from "./forms/v3";
+import { documentFormsV2Local } from "./forms/local";
+import { generateConfigFile } from "./generator";
+import { EmptyConfig, Forms } from "./types";
+
+const configFile: EmptyConfig = {
+  network: "",
+  wallet: {},
+  forms: [],
+};
+
+const formsV2: Forms[] = [{ version: "v2", forms: documentFormsV2 }];
```

Comment:
maybe a comment on why the need/constrain to mark the forms with their versions here, for `generateConfigFile` later?

Reply:
done, added into type file instead